src/OpenTelemetry.Exporter.Console/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 1.2.0-alpha2
+
+Released 2021-Aug-24
+
+* Add Histogram Metrics support.
```

Comment:
Consider adding the link(s) to the PR.

Reply:
Decided to not pollute changelog with too many entries, as we are in alpha stage.
Starting beta1 release, will add usual style changelogs

Reply:
Roger that, thanks!